Dockerfile

```diff
@@ -19,9 +19,18 @@ RUN go mod download
 COPY ./ ./
 
 # Build the executable to `/app`. Mark the build as statically linked.
-RUN CGO_ENABLED=0 go build \
-    -installsuffix 'static' \
-    -o /app ./cmd
+# Check if u
+ARG update="FALSE"
+RUN if [ $update = "TRUE" ] ;    \
+    then echo 'Building upd' ;   \
```

Comment:
what does 'upd' stand for?

Reply:
upd stands for update, the upd folder where another main package is placed